code-challenges.js

```diff
@@ -39,3 +90,23 @@ var numbersToAdd2 = [0, 7, -8, 12]
 // Expected output: [0, 7, -1, 11]
 var numbersToAdd3 = []
 // Expected output: []
+
+// create function that takes in an array
+// 
+
+
+const accumulatingSum = (array) => {
+    if (array.length === 0) {
+        return [];
+    }
+    let newArray = array.map(val => val)
+    return newArray.reduce(function(a, b) {
+        return a + b
+    })
+}
+
+console.log(accumulateSum(numbersToAdd1));
```

Comment:
the output for this function is not what the expected output is. right now it returns `60`. it would need to be refactored. also your console.log is calling your function but it's misspelled.